decidim-meetings/db/migrate/20220331064001_create_decidim_meetings_calendar_filters.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateDecidimMeetingsCalendarFilters < ActiveRecord::Migration[6.1]
+  def change
+    create_table :decidim_meetings_calendar_filters do |t|
```

Comment:
I don't like the idea of making this specific to the meetings component.

Sooner or later we will need short URLs somewhere else too.

Could we do something similar to what the [`shortener`](https://github.com/jpmcgrath/shortener) gem is doing. We could take inspiration from that. It seems actively developed right now but it's also not super popular, so it may stall at some point (within few years time). So, I'll leave that for your consideration.

Reply:
Can you also test if it works with the calendar if the URL is a 301 redirect?

E.g. what happens if you pass the current long link to bit.ly and pass that to the calendar program?

Otherwise we need to take it into account whether the short link should be an alternative render URL or a 301 redirect.

Reply:
@alecslupu have you seen this last comment? Do you have any thoughts? Thanks!

Reply:
@alecslupu Do you have time to check this issue? We need to get this issue resolved for 0.27, so we can also take over if you are busy.

Reply:
@ahukkanen , i already spoke with @andreslucena  about this, and I estimate that I will be able to have a try it next week.

Reply:
> Can you also test if it works with the calendar if the URL is a 301 redirect?

I've tried this along with @carolromero and this works. 

We've shortened with bitly a calendar from nightly. It worked well with Google Calendar, Outlook in Office 365 and macOS calendar software. If you want to try it yourselves: https://bit.ly/3l14K8D, there are some meetings at 2022/04/27.

Reply:
@alecslupu a friendly reminder about this issue 😄 

At the moment, this has a high priority for us, as this is blocking the v0.27.0 release. If you can handle these few days, that'd be great, but if not, don't sweat it, and we'll take this PR over on Monday. 

Thanks for your amazing work!!